test/controllers/RevaluationRateControllerSpec.scala

```diff
@@ -195,7 +195,7 @@ class RevaluationRateControllerSpec extends PlaySpec with OneServerPerSuite with
               RevaluationRate(None)
             )
             val result = TestRevaluationRateController.post(FakeRequest().withJsonBody(postData))
-            contentAsString(result) must include(Messages("gmp.error.reason.mandatory"))
```

Comment:
Can you now delete the message key: gmp.error.reason.mandatory ?

Reply:
it is used in InflationProofForm and DateOfLeavingForm